docker/docker.mk

```diff
@@ -110,15 +110,20 @@ ifeq ($(ONLY_DAPR_IMAGE),true)
 else
 	$(DOCKER) build --build-arg PKG_FILES=* -f $(DOCKERFILE_DIR)/$(DOCKERFILE) $(BIN_PATH) -t $(DOCKER_IMAGE_TAG)-$(TARGET_OS)-$(TARGET_ARCH)
 	test -f "$(BIN_PATH)/daprd" && \
-		$(DOCKER) build --build-arg PKG_FILES=daprd -f $(DOCKERFILE_DIR)/$(DOCKERFILE) $(BIN_PATH) -t $(DAPR_RUNTIME_DOCKER_IMAGE_TAG)-$(TARGET_OS)-$(TARGET_ARCH)
+		$(DOCKER) build --build-arg PKG_FILES=daprd -f $(DOCKERFILE_DIR)/$(DOCKERFILE) $(BIN_PATH) -t $(DAPR_RUNTIME_DOCKER_IMAGE_TAG)-$(TARGET_OS)-$(TARGET_ARCH) || \
```

Comment:
I just saw your message on Teams @shivamkm07 and I understand the problem now.

I am still not a fan of `|| true` because it would mask errors if the image fails to build.

How about:

```sh
if [ -f "$(BIN_PATH)/daprd" ]; then \
		$(DOCKER) build --build-arg PKG_FILES=daprd -f $(DOCKERFILE_DIR)/$(DOCKERFILE) $(BIN_PATH) -t $(DAPR_RUNTIME_DOCKER_IMAGE_TAG)-$(TARGET_OS)-$(TARGET_ARCH)
fi
```

This way it should not return a non-0 status code if the `test -f` fails.

Reply:
Done